tutorials/notebooks/README.md

```diff
@@ -0,0 +1,140 @@
+  | [YoloV8-nano](tutorials/notebooks/keras/ptq/example_keras_yolov8n.ipynb)              | &#x2705; PTQ                                                                                        |
+  | [EfficientDetLite0](tutorials/notebooks/keras/ptq/example_keras_effdet_lite0.ipynb) | &#x2705; PTQ <br/> &#x2705; [sony-custom-layers](https://github.com/sony/custom_layers) integration |
+
+</details>
+
+<details id="keras-gptq">
+  <summary>Gradient-Based Post-Training Quantization</summary>
+
+  | Tutorial                     | Included Features       |
+  |------------------------------|---------------|
+  | [MobileNetV2](tutorials/notebooks/keras/gptq/example_keras_mobilenet_gptq.ipynb)           | &#x2705; GPTQ |
+
+</details>
+
+<details id="keras-qat">
+  <summary>Quantization-Aware Training</summary>
+  
+  | Tutorial                                                                          | Included Features      |
+  |-----------------------------------------------------------------------------------|--------------|
+  | [QAT on MNIST](tutorials/notebooks/keras/gptq/example_keras_mobilenet_gptq.ipynb) | &#x2705; QAT |
+
+</details>
+
+
+<details id="keras-pruning">
+  <summary>Structured Pruning</summary>
+
+  | Tutorial                                                            | Included Features          |
+  |---------------------------------------------------------------------|------------------|
+  | [Fully-Connected Model Pruning](tutorials/notebooks/keras/pruning/example_keras_pruning_mnist.ipynb) | &#x2705; Pruning |
+
+</details>
+
+<details id="keras-export">
+  <summary>Export Quantized Models</summary>
+
+  | Tutorial                                                                              | Included Features         |
+  |---------------------------------------------------------------------------------------|-----------------|
+  | [Exporter Usage](tutorials/notebooks/keras/export/example_keras_export.ipynb) | &#x2705; Export |
+  
+</details>
+
+<details id="keras-debug">
+  <summary>Debug Tools</summary>
+
+  | Tutorial                                                                            | Included Features       |
+  |-------------------------------------------------------------------------------------|-------------------------|
+  | [Network Editor Usage](tutorials/notebooks/keras/debug_tools/example_keras_network_editor.ipynb) | &#x2705; Network Editor |
+  
+</details>
+
+## Pytorch Tutorials
+
+
+<details id="pytorch-quickstart-torchvision">
+  <summary>Quick-Start with Torchvision</summary>
+  
+  | Tutorial                                                                                                        |
+  |-----------------------------------------------------------------------------------------------------------------|
+  | [Quick Start - Torchvision Pretrained Model](tutorials/notebooks/pytorch/example_quick_start_torchvision.ipynb) |
+
```

Comment:
No features?

Reply:
No, it does not demonstrate any specific feature.